packages/wrangler/src/https-options.ts

```diff
@@ -1,5 +1,5 @@
 import * as fs from "node:fs";
-import { homedir, networkInterfaces } from "node:os";
+import os from "node:os";
```

Comment:
which one are we preferring as a team standard? I keep seeing destructuring in some places and default in others?

Reply:
In this case, see https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/__tests__/helpers/run-in-tmp.ts#L25-L29